apps/web/screens/Search/Search.tsx

```diff
@@ -201,12 +206,18 @@ const Search: Screen<CategoryProps> = ({
       News &
       AdgerdirPage &
       SubArticle &
-      OrganizationSubpage
+      OrganizationSubpage &
+      LinkItem
   >).map((item) => ({
     title: item.title,
     parentTitle: item.parent?.title,
     description: item.intro ?? item.description ?? item.parent?.intro,
-    link: linkResolver(item.__typename, item?.url ?? item.slug.split('/')),
+    link: linkResolver(
```

Comment:
Basically
```ruby
if typename == 'Link' then
    resolve as 'linkurl'
else
    resolve as either url, slug as applicable
```

Reply:
wouldn't it be better to leave graphql-type checking stuff outside of the component.. but looking at the rest of it, maybe we're a bit beyond that atm.

If I understand correctly, the item could be a link type or it could be a "normal link" .. and we want to handle both cases?

It would probably be nice then to make a helper for creating this link resolver.

Reply:
Yeah, actually. Maybe simply declaring the typename to be 'linkurl' in the importer and skipping this nonsense altogether.

Either way I should clean this up a bit

Reply:
Graphql doesn't like it when I lie to it :)
Moved special case to linkResolver similar to `linkurl`.